### ISSUE TITLE
add an option to show full commit message instead of short title

### ISSUE THREAD

**Issue by openaphid** [integrations/slack#405]
https://github.com/integrations/slack/blob/master/lib/messages/push.js#L18

Is it possible to show full commit msg by adding a slack cmd? It would be more convenient for my team which enforces detailed commit log.

**Comment by fmwr** [comment]
What do you mean by: "by adding a slack cmd"?
1. add a possibility to configure an option (by a command or whatever) that would show full commit message 
2. show full commit message only after typing a another command

I hope you mean 1 - which I'm also for.

**Comment by openaphid** [comment]
The first one:)

**Comment by fmwr** [comment]
Definitely relevant.

Why `wontfix` label added?

**Comment by unforswearing** [comment]
Just added slack integration and would love to have the option to see full commit messages, not just the title/subject when using `/github subscript owner/repo commits`.

**Comment by scepticulous** [comment]
Thanks for bringing this topic back to our attention. 

I am re-opening it, but I can't give any details if and when we are going to take a look at this.

🔈Everybody who thinks this is important or valuable to him or her, please upvote the OP, so that we can get an indicator about the existing demand.  🔈 
Thanks 🙇

**Comment by preciselytom** [comment]
This would improve communication within our team. We are encouraged to use a short title and detailed body, rather than a long title, but Slack only shows the title.

**Comment by basileFourcade** [comment]
It would be great to be able to show the full description in Slack.
In the specific case that I'm interested in, it will allow people without a github access to read "more than one line information" in a slack channel. (if they had access, it would be to fight lazyness to click on the commit link)
Thanks

**Comment by Zoinen** [comment]
very needed feature, please reconsider its priority. I'd even say it should be the default behavior

**Comment by runik** [comment]
would be awesome to have this feature really needed

**Comment by jlichti** [comment]
we'd love to have this as an option as well. bitbucket behaved this way and not many of our descriptions are getting cut off

**Comment by juliandreas** [comment]
This feature is really needed within our team

**Comment by joeydong** [comment]
+1

**Comment by stephenasamoah** [comment]
We are still rating. Please consider implementing this feature.

**Comment by hdavis91** [comment]
Any updates on this? It would be very useful for our team as well.

**Comment by natebluewizard** [comment]
Adding a comment to chime in -- seeing the full description with a commit would be very helpful for our team as well!

**Comment by james-koehne** [comment]
This would be a great feature for our team.

**Comment by bluewizardsupport** [comment]
Another voice behind this option - had to remove it almost as soon as we installed it ;-( Sadge.

**Comment by AndyWiltshireNZ** [comment]
Seeing the full description of a commit is 'Essential' - with a capital 'E' - much love for the developer, but please can we have this feature - thanks in advance! :)

**Comment by hugoup** [comment]
What status on this, the commit-integration is kinda pointless if i still have to open github and read the body of the commit.

**Comment by dariusk** [comment]
Adding this functionality as a configurable option would really help encourage developers to write longform descriptive commit messages, and would make the notifications feel like a substantive communication and less like spam, at least in my org.

**Comment by cloutsocks** [comment]
would also love this

**Comment by beddari** [comment]
Another vote, fixing this would improve communication a lot.

**Comment by nigel-loops** [comment]
Voting this. Cant believe this hasnt been done based on all the requests and how easy this is to implement via slack dev toolkit / webhook

**Comment by khcpietro** [comment]
4 years passed.. please listen our voices

**Comment by KuzubValentin** [comment]
Pretty disappointed that this is not implemented. It is very inconvenient.

**Comment by jon-minepi** [comment]
requesting this feature!

**Comment by JordanKDesigns** [comment]
+1000

Found this issue via Google, saw it was created in 2018 and was hopeful as I scrolled down it would resolved.

Now I'm sad.

**Comment by nickdawes** [comment]
Would like to see this feature too :)

**Comment by kellybarber** [comment]
I would also like to see this feature in 2024

**Comment by ronenbjorkquist** [comment]
Would like to see this too, but probably not since this issue started in 2018...

**Comment by hpatchett** [comment]
Commenting to bump this issue! Our team would love the ability to configure this. 

Not really sure why the description / body of the commit is omitted in the first place?

**Comment by PierreNicoletti** [comment]
Would love to have this feature as well !
